docs/docs/advanced/private-registries/docker-hub.md

```diff
@@ -1,7 +1,2 @@
-Docker Hub needs `TRIVY_USERNAME` and `TRIVY_PASSWORD`.
-You don't need to set ENV vars when download from public repository.
-
-```bash
-export TRIVY_USERNAME={DOCKERHUB_USERNAME}
```

Comment:
Is this specific to DockerHub? Why are we calling it TRIVY_USERNAME and PASSWORD and not DOCKER_USERNAME and PASSWORD?

Reply:
Or instead REGISTRY_USERNAME

Reply:
That's why I removed it from Docker Hub and moved it under private images 😄

Reply:
No, it is not docker specific. I renamed them.
https://github.com/aquasecurity/trivy/pull/3906/files#diff-a812993c8d91c3eacbf5538b3f3cffaa8a1fe76c48f493caf7882484113d648fR8